exporters/zipkin/zipkin_test.go

```diff
@@ -364,3 +366,20 @@ func TestErrorOnExportShutdownExporter(t *testing.T) {
 	assert.NoError(t, exp.Shutdown(context.Background()))
 	assert.NoError(t, exp.ExportSpans(context.Background(), nil))
 }
+
+func TestLogrFormatting(t *testing.T) {
+	format := "string %q, int %d"
+	args := []interface{}{"s", 1}
+
+	var buf bytes.Buffer
+	l := funcr.New(func(prefix, args string) {
+		_, _ = buf.WriteString(fmt.Sprint(prefix, args))
+	}, funcr.Options{})
+	exp, err := New("", WithLogr(l))
+	require.NoError(t, err)
+	exp.logf(format, args...)
```

Comment:
Just adding a blank line to have the "act" section of the unit test more visible.

```suggestion

	exp.logf(format, args...)
```